src/Aspire.Dashboard/README.md

```diff
@@ -63,6 +63,10 @@ Set `Dashboard:Frontend:AuthMode` to `OpenIdConnect`, then add the following con
 - `Dashboard:Frontend:OpenIdConnect:RequiredClaimType` specifies the (optional) claim that be present for authorized users. Defaults to empty.
 - `Dashboard:Frontend:OpenIdConnect:RequiredClaimValue` specifies the (optional) value of the required claim. Only used if `Dashboard:Frontend:OpenIdConnect:RequireClaimType` is also specified. Defaults to empty.
 
+#### Memory limits
+
+- `Dashboard:Frontend:ConsoleLogHistoryLimit` specifies the (optional) maximum number of console log messages to keep in memory. Defaults to 10,000. When the limit is reached, the oldest messages are removed.
```

Comment:
The telemetry limits configuration have this naming standard: MaxLogCount

I think this should be called MaxConsoleLogCount to follow that standard.